seismic-app/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_GOOGLE_MAPS_KEY=API_KEY_GOES_HERE
```

Comment:
also. an api key is needed

Reply:
@RaduCStefanescu will be able to provide a key, just that first we'll need to move to Here maps